### DISCUSSION TITLE
Video Support

### DISCUSSION THREAD

**Discussion by JustinianErdmier** [coenm/ImageHash#123] [Q&A]
Is it possible to use this library with .mp4 files as well? If not, how difficult would it be to learn and contribute to the project to give such capability?

**Comment by coenm** [comment]
No, using this library, it is not possible and I don't see how it could be implemented. I'm willing to accept PR's if you managed to implement such feature. For now, I suggest to do some research in the field video hashing. 

Sorry I wasn't able to assist your
